e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -88,19 +82,15 @@ func (s *IbcEurekaTestSuite) SetupSuite(ctx context.Context) {
 		err := os.Chdir("../..")
 		s.Require().NoError(err)
 
-		s.key, err = crypto.GenerateKey()
-		s.Require().NoError(err)
-		testKeyAddress := crypto.PubkeyToAddress(s.key.PublicKey).Hex()
-
-		s.deployer, err = eth.BuildWallet(ctx, "deployer", "")
+		s.key, err = eth.CreateAndFundUser()
+		fmt.Println(err)
+		time.Sleep(5 * time.Minute)
```

Comment:
Is this line really needed? And even if it is needed, shouldn't it be behind an if case for POS?

Reply:
I've removed this along with other improvements. We can merge if you agree